Functionality to login, sign up, and today buttons

pfishe13 opened the pull request:
Functionality to login, sign up, and today buttons. Opens blank pages.

pfishe13 committed:
Functionality to login, sign up, and today buttons

Functionality to login, sign up, and today buttons. Opens blank pages.

amoor131 reviewed (dismissed):
the buttons do not currently link to the correct pages. change where they link to and I'll review this branch again.

pfishe13 committed:
Added calories template file

Added calories template file. This will be called when the CALORIES button is click from the banner.

pfishe13 committed:
Created overview template page

Created overview template page. This will be called when the OVERVIEW button is clicked from the banner.

pfishe13 committed:
Added routes to the calories and overview pages

Calories is /calories and overview is /overview

pfishe13 committed:
Added functionality to calories and overview button.

'Log Out' and 'Dark Mode' button have yet to be implemented

amoor131 reviewed (approved):
This works great! I'm not sure if I'm meant to be the one to merge so I'll just leave my approval for now.

moha-ali001 commented:
works and merged
